### PR TITLE
BNF MALZ IV

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Minor changes in carious mss,
the first item from Alaqma Maryam inventory.

### CODE REVIEW

**Review by thea-m — APPROVED** [review]

File: ES/ESmy023.xml [hunk 1]
```diff
@@ -276,8 +276,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <material key="parchment"/>
                         </support>
                         <extent><measure unit="leaf" quantity="91">91</measure>
-                              <measure unit="leaf" type="blank"/>
-                              <locus target="#1rv #2rv #3rv #91rv"/>
+                              <measure unit="page" type="blank"/>
```
Comment (eu-genia) [inline ES/ESmy023.xml:279]: do we use page here? I thought we count in pages only for paginated mss

File: ParisBNF/et/BNFet343.xml [hunk 2]
```diff
@@ -25,9 +25,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <collection>Manuscrits orientaux</collection>
                         <collection>Fonds éthiopien</collection>
                         <collection>Griaule</collection>
-                        <idno>BnF Éthiopien 343 (Griaule 39)</idno>
+                        <idno>BnF Éthiopien 350 (Griaule 46)</idno>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet343.xml:28]: is this correct? the file is still called 343
if the description is for 350 the filename and xml:id should also be changed (?)

File: ParisBNF/et/BNFet343.xml [hunk 3]
```diff
@@ -38,14 +38,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                     <listBibl type="catalogue">
                                         <bibl>
                                             <ptr target="bm:Grebaut1941GriauleII"/>
-                                            <citedRange unit="page">20-25</citedRange>                                           
+                                            <citedRange unit="page">58-61</citedRange>                                           
                                         </bibl>
                            
                                     </listBibl>
-                                 <listRelation>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet343.xml:45]: I do not understand these changes. 
We had a stub called Griaule 343. It has been renamed to BNF et 343, with a relation
If the description is for BNF et 350 then the filename should be changed and the relation updated accordingly to `<relation name="betmas:formerlyAlsoListedAs" active="BNFet350" passive="BNFgriaule343"/>`

If et 350 is a new record then BNFet343 should stay as is and BNFet350 created

File: ParisBNF/et/BNFet343.xml [hunk 4]
```diff
@@ -4,9 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Vita and Miracles of Takla Hāymānot</title>
+                <title>Collection of malkǝʾ-hymns</title>
                 <editor role="generalEditor" key="AB"/>
-                <editor key="DR"/>
+                <editor key="DN"/>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet343.xml:9]: ??

File: ParisBNF/et/BNFet343.xml [hunk 5]
```diff
@@ -68,14 +65,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    <term key="Poetry"/>
                 </keywords>
             </textClass>
             <langUsage><language ident="en">English</language></langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="DR" when="2017-06-09">Created entity</change>
-            <change who="DR" when="2023-10-09">Added relation to deleted BNFgriaule343</change>
-        </revisionDesc>
+            <change who="DR" when="2024-04-04">Created entity</change>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet343.xml:74]: ??

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy023.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy023.xml:279]: We do; I changed back, but there was a consideration behind, maybe to discuss separately.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**